components/EventDetails.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import { DateBlock } from '../components';
+import { Link } from '../routes';
+
+const renderAddress = ({ address_lines: addressLines, locality, region, postal_code: postalCode }) => {
+  // Will have to see how this data structure holds up over different events
+  return (
+    <div>
+      { addressLines[0] && <span>{addressLines[0]}</span> }
+      <span>{locality} {region}, {postalCode}</span>
+    </div>
+  );
+};
+
+const renderTimeRange = (startDate, endDate) => {
+  // TDB how to properly format this - would be nice to avoid bringing in moment.js
```

Comment:
+1 for Moment, as it is an industry-standard. @ignu - with regards to the mutation of the dates, couldn't we just write our own functions using Moment that output a new date rather than mutating the date we're processing and get the same result?